src/types/Options.ts

```diff
@@ -60,6 +60,12 @@ export type NamespaceResolver = (prefix: string) => string | null;
  * @public
  */
 export type Options = {
+	/**
+	 * Whether or not the AST should get annotated after parsing an expression. The annotation adds
+	 * additional type information to the AST.
+	 */
```

Comment:
Can you add a `@beta` tag to this? So that we can remove the opt-in flag later on?